src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,532 @@
+        name=get_mnemonic(source_uniprot_id),
+    )
+    target = pybel.dsl.Protein(
+        namespace='uniprot',
+        identifier=target_uniprot_id,
+        name=get_mnemonic(target_uniprot_id),
+    )
+
+    for pubmed_id in pubmed_ids:
+
+        # INCREASE
+
+        if relation in INTACT_INCREASES_ACTIONS:
+
+            # oxidoreductase activity
+            if relation == 'oxidoreductase activity electron transfer reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification('Red'),
+                )
+
+                graph.add_increases(
+                    source,
+                    target_mod,
+                    citation=pubmed_id,
+                    evidence=EVIDENCE,
+                    subject_modifier=pybel.dsl.activity(),
+                )
+                continue
+            # isomerase reaction
+            elif relation == 'isomerase reaction' \
+                    or relation == 'isomerization  reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='isomerase activity',
+                        namespace='GO',
+                        identifier='0016853',
+                    ),
+                )
+
+                graph.add_increases(
+                    source,
+                    target_mod,
+                    citation=pubmed_id,
+                    evidence=EVIDENCE,
+                    subject_modifier=pybel.dsl.activity(),
+                )
+                continue
+            # proline isomerase reaction
+            elif relation == 'proline isomerization  reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='protein peptidyl-prolyl isomerization',
+                        namespace='GO',
+                        identifier='0000413',
+                    ),
+                )
+
+                graph.add_increases(
+                    source,
+                    target_mod,
+                    citation=pubmed_id,
+                    evidence=EVIDENCE,
+                    subject_modifier=pybel.dsl.activity(),
+                )
+                continue
+            # protein amidation
+            elif relation == 'amidation reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='protein amidation',
+                        namespace='GO',
+                        identifier='0018032',
+                    ),
+                )
+            # protein deamination (negative effect, increases the deaminated target protein abundance)
+            elif relation == 'deamination reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='protein deamination',
```

Comment:
would be better to assert a decreases relationship (opposite polarity) and use protein amination as the modification

Reply:
I agree that in this case it seems bizarre to term is as a 'increases' relation. The reason for why I did it is that there is no exact opposite term in GO like 'protein amination'. What I would suggest instead is to use the term 'amino acid binding' [GO: 0016597](https://www.ebi.ac.uk/QuickGO/term/GO:0016597). In this way, we can use it as a 'decreases' relation,

Reply:
Closed here: 3da5c2297ec1e9b28ca0119ecda2fa5db21df0ac